src/drive/targets/browser/setupAppContext.js

```diff
@@ -12,9 +12,84 @@ import registerClientPlugins from 'drive/lib/registerClientPlugins'
 
 import appMetadata from 'drive/appMetadata'
 import configureStore from 'drive/store/configureStore'
-import { schema } from 'drive/lib/doctypes'
+import { schema, DOCTYPE_FILES } from 'drive/lib/doctypes'
 import { Document } from 'cozy-doctypes'
 import { hashHistory } from 'react-router'
+import {
+  getAdapterPlugin,
+  getOldAdapterName
+} from '../../mobile/lib/cozy-helper'
+import { isIOSApp, isMobileApp } from 'cozy-device-helper'
+import PouchLink from 'cozy-pouch-link'
+import {
+  buildSuggestionQuery,
+  buildRecentQuery,
+  buildDriveQuery,
+  buildFolderQuery
+} from 'drive/web/modules/queries'
+
+/**
+ * Init Client
+ *
+ * @param {string} cozyUrl - Url
+ * @param {string} token - token
+ * @returns {CozyClient}
+ */
+const initClient = (cozyUrl, token) => {
+  const stackLink = new StackLink()
+  const adapter = getOldAdapterName()
+
+  const pouchLinkOptions = {
+    doctypes: [DOCTYPE_FILES],
+    doctypesReplicationOptions: {
+      [DOCTYPE_FILES]: {
+        strategy: 'fromRemote',
+        warmupQueries: [
+          buildDriveQuery({
+            currentFolderId: 'io.cozy.files.root-dir',
+            type: 'directory',
+            sortAttribute: 'name',
+            sortOrder: 'asc'
+          }),
+          buildRecentQuery(),
+          buildFolderQuery('io.cozy.files.root-dir'),
+          buildSuggestionQuery()
+        ]
+      }
+    },
+    pouch: {
+      plugins: [getAdapterPlugin(adapter)],
+      options: {
+        adapter,
+        location: 'default'
+      }
+    },
+    initialSync: true
+  }
+
+  // TODO: is it useful?
+  // if (isMobileApp() && isIOSApp()) {
+  //   pouchLinkOptions.pouch = {
+  //     plugins: [getAdapterPlugin(adapter)],
+  //     options: {
+  //       adapter,
+  //       location: 'default'
+  //     }
+  //   }
+  // }
+
+  const pouchLink = new PouchLink(pouchLinkOptions)
+
+  return new CozyClient({
+    uri: cozyUrl,
+    token: token, // TODO: NEEDED? Previously used to login https://github.com/cozy/cozy-client/blob/cfeec8b59893d6083aa9b814f895be774a7f8693/packages/cozy-client/src/CozyClient.js#L219
+    // oauth: getOauthOptions(), // TODO: NEW
```

Comment:
this code inspired by mobile/cozy-helpers, keeps using token (and not oauth).

I wonder why, in the process, I am facing 2 main issues of invalid JWT token:
- one when realtime is activating
- a second one when replicating the pouchdb (fetchRemoteInstance) during this [fetch](https://github.com/cozy/cozy-client/blob/90125ce0f248e54a50776481b3034ecd97836664/packages/cozy-pouch-link/src/remote.js#L23) called by [startReplication](https://github.com/cozy/cozy-client/blob/a808320d962ab5a69bb67adbba2bf587e110aad9/packages/cozy-pouch-link/src/startReplication.js#L172)